pkg/storage/cacher_test.go

```diff
@@ -299,23 +298,3 @@ func TestFiltering(t *testing.T) {
 	verifyWatchEvent(t, watcher, watch.Modified, podFooPrime)
 	verifyWatchEvent(t, watcher, watch.Deleted, podFooPrime)
 }
-
-/* TODO: So believe it or not... but this test is flakey with the go-etcd client library
```

Comment:
A replica of this test exists at the etcd layer and is enabled now.  I removed b/c it seemed redundant.